### ISSUE TITLE
chord broken for customized queue name

### ISSUE THREAD

**Issue by jianbin-wei** [celery/celery#2085]
With following code, I want to put the task in "new_queue" task queue but it gets into "celery" queue.  It works on 3.0.23 but is broken with 3.1.12.

``` python
task_group = []
task_options = {"dirty_bucket": dirty_bucket}
task_group.append(report_task.s(task_options=task_options))
job_chord = chord(task_group, callback.s())
chord_options = {"queue": "new_queue",
                            "retry": True,
                            "retry_policy": {"max_retries": 5}
                            }
job_chord.apply_async(args=[], kwargs={}, **chord_options)
```

**Comment by porn** [comment]
Sorry, but in which version is this fixed? I can't see the 3.2 branch...

**Comment by yingzong** [comment]
Not only is there no 3.2 branch, this is also not fixed as of v3.1.18. The current latest version is v3.1.20. The answer is that the super-task (whether group, chord, or chunk) goes into the default queue.
